syslog_rfc5424_formatter.py

```diff
@@ -73,8 +92,41 @@ def format(self, record):
             isotime = isotime + 'Z'
 
         record.__dict__['isotime'] = isotime
+        record.__dict__['procid'] = self.procid if self.procid else '-'
```

Comment:
this is a breaking change; why not have it default to the `process` key from the `logging` module (which is always supposed to be the current PID, which is what goes into this field)?

Reply:
I have update to default to current value (PID).